atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowQueryFactory.java

```diff
@@ -52,8 +49,10 @@ public boolean hasOverflowValues() {
         for (OverflowValue overflowId : overflowIds) {
             oraRows.add(new Object[] { null, null, overflowId.id });
         }
-        ArrayHandler arg = jdbcHandler.createStructArray("PT_MET_CELL_TS", "PT_MET_CELL_TS_TABLE", oraRows);
-        switch (migrationState) {
+        ArrayHandler arg = config.jdbcHandler().createStructArray(
+                config.shared().tablePrefix() + "CELL_TS",
```

Comment:
nit/question: any reason this was all caps before, going to look a little weird as a_CELL_TS...